test/test_grid.py

```diff
@@ -196,8 +197,63 @@ def test_integrate(self):
 
 class TestFaceAreas(TestCase):
 
+    def test_calculate_total_face_area_triangle(self):
+        """Create a uxarray grid from vertices and calculate the area using
+        gaussian and triangular quadrature rules."""
+
+        verts = np.array([[0.57735027, -5.77350269e-01, -0.57735027],
+                          [0.57735027, 5.77350269e-01, -0.57735027],
+                          [-0.57735027, 5.77350269e-01, -0.57735027]])
+        vgrid = ux.Grid(verts)
+
+        # get node names for each grid object
+        x_var = vgrid.ds_var_names["Mesh2_node_x"]
+        y_var = vgrid.ds_var_names["Mesh2_node_y"]
+        z_var = vgrid.ds_var_names["Mesh2_node_z"]
+
+        vgrid.ds[x_var].attrs["units"] = "m"
+        vgrid.ds[y_var].attrs["units"] = "m"
+        vgrid.ds[z_var].attrs["units"] = "m"
```

Comment:
Using the variable name dictionary here isn't necessary. 

```Python
vgrid.Mesh2_node_x.attrs["units"] = "m"
vgrid.Mesh2_node_y.attrs["units"] = "m"
vgrid.Mesh2_node_z.attrs["units"] = "m"

Reply:
Right, waiting for the new design for updating this old PR.